hassio/addons/utils.py

```diff
@@ -43,6 +43,10 @@ def rating_security(addon: AddonModel) -> int:
     if addon.access_auth_api:
         rating += 1
 
+    # Home Assistant Protect WebUI
+    if addon.with_ingress:
```

Comment:
Second thought, let's combine this with the `access_auth_api`, so adding both still doesn't get you +3

```suggestion
    if addon.with_ingress or addon.access_auth_api:
```

Reply:
make sense

Reply:
or make ingress +2?

Reply:
Sure that is fine as well, that adds additional security.

```py
if addon.with_ingress or addon.access_auth_api:
   rating += 1

if addon.with_ingress
   rating += 1
```

Gives +2 for ingress, +1 for auth api, but +2 for both (instead of +3)